src/ragas/metrics/_summarization.py

```diff
@@ -232,7 +235,9 @@ async def _get_questions(
             result_text, p_value, self.llm, self.max_retries
         )
         
-        assert response and response.questions, "No questions generated, unable to calculate the score."
+        if not response or not response.questions:
+            logging.error("No questions generated, unable to calculate the score.")
+            return math.nan
```

Comment:
```suggestion
            return np.nan
```

Reply:
Sure, I have changed the all and made a new commit.